src/ode_solver/explicit_ode_solver.cpp

```diff
@@ -65,7 +65,9 @@ void RungeKuttaODESolver<dim,real,n_rk_stages,MeshType>::step_in_time (real dt,
             this->rk_stage[i] = solver.current_solution_estimate;
 
         } // u_n + dt * sum(a_ij * k_j) <explicit> + dt * a_ii * u^(i) <implicit>
-            
+        
+        compute_stored_quantities(i);
```

Comment:
Could we consider changing the name of this function to something more descriptive?

Reply:
For sure. For now I will rename this to `store_stage_solutions()` and make the description in `explicit_ode_solver.h` clearer.